### ISSUE TITLE
Nicer with linters

### ISSUE THREAD

**Issue by julien-f** [robertklep/node-benchr#3]
Currently, my linter is reporting `suite` and `benchmark` are undefined.
I know I can configure it to ignore them but I would much prefer avoid any linter-specific configuration.

I see 2 potential answers:

1. make them importable
2. make it possible to export a single factory function which would receive them as (named) params

I can understand than some people might not like that (including you perhaps) and would consider this as boilerplate but IMHO, I consider globals as magical and confusing, and I dislike, as I said, adding linter-specific configuration in my code.

Have a nice day :)

**Comment by julien-f** [comment]
[Here is my own basic implementation](https://gist.github.com/julien-f/75ad03da30364d39bb10f57fd97da775).

But I would love to avoid creating a new *yet another* benchmark runner ;)

**Comment by robertklep** [comment]
I could make them importable, but all that would really do is similar to adding this to your own code:
```
const suite     = global.suite;
const benchmark = global.benchmark;
```

So not great. The idea of exporting a factory function from a test file is perhaps more clean, and easy enough to implement.

**Comment by robertklep** [comment]
Implemented with cdbc3f2115.

Before I push out a new release I'll take a look at #2 first, see if I can fix that fast and fold it into a new release with this one.

**Comment by robertklep** [comment]
Pushed v3.2.0
